lib/matplotlib/axes/_axes.pyi

```diff
@@ -318,7 +318,9 @@ class Axes(_AxesBase):
         normalize: bool = ...,
         hatch: str | Sequence[str] | None = ...,
         data=...,
-    ): ...
+    ) -> tuple[list[Wedge], list[Text]] | tuple[
+        list[Wedge], list[Text], list[Text]
+    ]: ...
```

Comment:
yeah, `boilerplate` is not (yet) set up to do `overload`s

I think it could be done, but the templating gets harder.

potentially it could be done manually and taken out of what `boilerplate` does, or we could look to improve `boilerplate`